js/test/beautify-javascript-tests.js

```diff
@@ -1277,6 +1277,15 @@ function run_javascript_tests(test_obj, Urlencoded, js_beautify, html_beautify,
             'a.b("c",\n' +
             '    () => d.e\n' +
             ')');
+        
+        // Issue 810 - es6 object literal detection
+        bt(
+            '{\n' +
+            '    a,\n' +
+            '    b: c,\n' +
+            '    d: e,\n' +
```

Comment:
You might as well put both inputs that you listed in the issue report in here.  They were just a little more interesting than this.